dashboard/src/components/diagram/right-sidebar/proto-components/variable-assignment.tsx

```diff
@@ -0,0 +1,11 @@
+import { getVariable } from "@/utils/data/variables";
+import { VariableAssignment } from "littlehorse-client/proto";
+
+// todo: more complex handling for runs, where other variableAssignment fields will be populated
+export function VariableAssignmentComponent(variableAssignment: VariableAssignment) {
+    return (
+        <div>
+            {getVariable(variableAssignment)}
+        </div>
+    )
+}
```

Comment:
Looks like you're either not running a formatter or you have disabled the end-of-file format. this is important so the next person that runs the formatter doesn't have to push unrelated changes.